appengine-skeleton-archetype/pom.xml

```diff
@@ -18,8 +18,8 @@
   </licenses>
   <developers>
     <developer>
-      <name>Takashi Matsuo</name>
-      <email>tmatsuo@google.com</email>
+      <name>Les Vogel</name>
+      <email>lesv@google.com</email>
```

Comment:
Adding you is the right thing. I'm ok with removing me, but I think it is natural to keep both.